src/snackbar/README.md

```diff
@@ -0,0 +1,63 @@
+# Snackbar
+Snackbars provide brief feedback about an operation through a message.
+
+## Usage
+
+### Api
+
+#### SnackbarService
+
+```ts
+import { SnackbarService, SnackbarRef } from "@ssv/au-ui";
+
+// inject it (within a class) e.g.
+constructor(
+    private snackbarService: SnackbarService
+)
+
+// open
+const snackbarRef = this.snackbarService.open(`Message sent!`);
+
+// open - with action + options
+const snackbarRef = this.snackbarService.open("You have been disconnected", "Reconnect", {
+    duration: 5000
+});
+
+// dismiss - active only
+this.snackbarService.dismiss();
+
+// dimissAll - queued and active
+this.snackbarService.dismissAll();
+```
+
+#### SnackbarRef
+Reference returned when opening a snackbar.
+
+```ts
+// onDismiss - subscribe on dismiss callback
+snackbarRef.onDismiss(() => {
+    // ...
+});
+
+// onAction - subscribe on action callback
+snackbarRef.onAction(() => {
+    // ...
+});
+
+// dismiss - dismisses the snackbar
+snackbarRef.dismiss();
+```
+
+## Global Configuration
+Options which can be configured globally.
+
+| Name        | Type         | Default |
+|-------------|--------------|---------|
+| duration    | number       | 3000    |
```

Comment:
true, only custom/complex types were being done with backticks. 

would you do everything or nothing?